### PR TITLE
Add 9.1.0 blogpost

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

File: blog/next-9-1-0-preview.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const meta = {
+  title: 'Next.js 9.1.0',
+  description: ``,
```
Comment (Timer) [inline blog/next-9-1-0-preview.mdx]: A little sales pitchy, but maybe:
```suggestion
  description: `Next.js 9.1 leverages convention to support more project structures by default. Upgrade today and leverage all Next.js 9 has to offer!`,
```

Or:
```suggestion
  description: `Next.js 9.1 leverages convention to support more project structures by default. Additionally, we're previewing new feature support such as CSS imports, advanced bundling, and static error pages.`,
```

File: pages/blog/next-9-1-0.mdx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+export const config = { amp: true }
+
+export default withPost({ ...meta })
+
+Today, we are excited to announce Next.js 9.1.0 with `src` and `public` directory support.
+
+**New in This Release**
+
+- **[`src` Directory Support](about:blank#gzip-compression-by-default)**: The `pages` directory can now be nested under the `src` folder, supporting a wider variety of application setups.
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[`src` Directory Support](#src-directory-support)**: The `pages` directory can now be nested under the `src` folder, supporting a wider variety of application setups.
```

File: pages/blog/next-9-1-0.mdx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+- **[`public` Directory Support](about:blank#typescript-report-on-active-pages-only)**: Define files to be mounted at the root of your application's URL (e.g. `favicon.ico`).
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[`public` Directory Support](#public-directory-support)**: Define files to be mounted at the root of your application's URL (e.g. `favicon.ico`).
```

File: pages/blog/next-9-1-0.mdx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+**Previewing in This Release**
+
+- **[Built-in CSS Support](about:blank#improved-concurrency-in-windows-environments)**: Applications can soon import Global CSS and leverage development hot reloading and advanced production optimizations, compilation, and polyfilling.
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[Built-in CSS Support](#built-in-css-support)**: Applications can soon import Global CSS and leverage development hot reloading and advanced production optimizations, compilation, and polyfilling.
```

File: pages/blog/next-9-1-0.mdx [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+- **[Static Error Pages](about:blank#telemetry)**: Statically export expected error pages (like 404) for better availability (CDN).
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[Static Error Pages](#static-error-pages)**: Statically export expected error pages (like 404) for better availability (CDN).
```

File: pages/blog/next-9-1-0.mdx [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+- **[Module / Nomodule](about:blank#improved-nexthead-element-tracking)**: Ship smaller JavaScript bundles to end-users running on evergreen browsers.
+- **[Improved Bundle Splitting](about:blank#preventing-non-pages-in-the-pages-directory)**: Ship fewer bytes to your end-user, improving TTI and page transition speed. Large library chunks are also long-term cached across deploys.
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[Improved Bundle Splitting](#improved-bundle-splitting)**: Ship fewer bytes to your end-user, improving TTI and page transition speed. Large library chunks are also long-term cached across deploys.
```

File: pages/blog/next-9-1-0.mdx [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+**New in This Release**
+
+- **[`src` Directory Support](about:blank#gzip-compression-by-default)**: The `pages` directory can now be nested under the `src` folder, supporting a wider variety of application setups.
+- **[`public` Directory Support](about:blank#typescript-report-on-active-pages-only)**: Define files to be mounted at the root of your application's URL (e.g. `favicon.ico`).
+
+**Previewing in This Release**
+
+- **[Built-in CSS Support](about:blank#improved-concurrency-in-windows-environments)**: Applications can soon import Global CSS and leverage development hot reloading and advanced production optimizations, compilation, and polyfilling.
+- **[Static Error Pages](about:blank#telemetry)**: Statically export expected error pages (like 404) for better availability (CDN).
+- **[Module / Nomodule](about:blank#improved-nexthead-element-tracking)**: Ship smaller JavaScript bundles to end-users running on evergreen browsers.
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
- **[Module / Nomodule](#module--nomodule)**: Ship smaller JavaScript bundles to end-users running on evergreen browsers.
```

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/blog/next-9-1-0**](https://next-site-git-fork-timneutkens-add-next-upcoming-features-blog.zeit.now.sh/blog/next-9-1-0) |
|:-:|
|<a href="https://next-site-git-fork-timneutkens-add-next-upcoming-features-blog.zeit.now.sh/blog/next-9-1-0">Screenshot of /blog/next-9-1-0</a><br /><sup><a href="https://microlink-cdn.s3.amazonaws.com/s/5PZ9ktPxbeojqA/9zGdnhHLZRuTU_">(view full size)</a> |







Commit 0f4628d7b09b0889ceab3d8da1475cabe6665366 (https://next-site-git-fork-timneutkens-add-next-upcoming-features-blog.zeit.now.sh).

File: blog/next-9-1-0-preview.mdx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+      avatar: '/static/team/jj.jpg',
+      twitter: '_ijjk'
+    },
+    {
+      name: 'Luis Alvarez',
+      avatar: '/static/team/lfades.jpg',
+      twitter: 'luis_fades'
+    }
+  ]
+}
```
Comment (Timer) [inline blog/next-9-1-0-preview.mdx]:
```suggestion
}

Today, we are excited to announce Next.js 9.1.0 with `src` and `public` directory support.

**New in This Release**

- [`src` Directory Support](#src-directory-support)
- [`public` Directory Support](#public-directory-support)

**Previewing in This Release**

- [Built-in CSS Support](#built-in-css-support)
- [Static Error Pages](#static-error-pages)
- [Module / Nomodule](#module--nomodule)
- [Improved Bundle Splitting](#improved-bundle-splitting)
```

File: pages/blog/next-9-1-0.mdx [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+
+For example, currently, Next.js has to provide polyfills for async/await syntax as code might be executed in browsers that do not support async/await which would break.
+
+To avoid breaking older browsers while sending modern JavaScript to browsers that support newer syntax Next.js will utilize the [module/nomodule](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/) pattern. The module/nomodule pattern provides a reliable mechanism for serving modern JavaScript to modern browsers while allowing older browsers to fall back to polyfilled ES5 code.
+
+This new feature is currently being tested in production by multiple large-scale Next.js applications to collect real-world data. The results of these tests are looking promising and more details on the performance improvements of this change will be shared soon.
+
+One example of initial results we have seen is this comparison by [Natalie Marleny](https://twitter.com/NatalieMarleny) about the bundle sizes before and after enabling built-in module/nomodule support in Next.js:
+
+EMBED:
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
```

File: pages/blog/next-9-1-0.mdx [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
```

File: pages/blog/next-9-1-0.mdx [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+[https://twitter.com/timneutkens/status/1162845290015199232](https://twitter.com/timneutkens/status/1162845290015199232)
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
<ReactTweet id="1162845290015199232" />
```

File: pages/blog/next-9-1-0.mdx [hunk 12]
```diff
@@ -0,0 +1,169 @@
+import withPost from '../../components/blog/with-post'
+import Image from '../../components/image'
+import { meta } from '../../blog/next-9-1-0-preview.mdx'
+import { TerminalInput } from '../../components/text/terminal'
```
Comment (Timer) [inline pages/blog/next-9-1-0.mdx]:
```suggestion
import { TerminalInput } from '../../components/text/terminal'
import ReactTweet from 'react-tweet-embed'
```

**Pull request merged**